src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -579,6 +579,11 @@ public static void Delete(String path, bool recursive)
             String fullPath = Path.GetFullPath(path);
             FileSystem.Current.RemoveDirectory(fullPath, recursive);
         }
+
+        public static string[] GetLogicalDrives()
```

Comment:
Every other piece of functionality in this library that needs to be specialized by platform goes through the FileSystem abstraction... except for this one new one you're adding. This is not different enough to warrant that departure, it's confusing for someone familiar with the pattern employed in the remainder of the library, it doesn't result in any  significant additional code, etc. I do not understand the resistance here to staying consistent with everything else in the library.